### ISSUE TITLE
OpenID/OAuth and Single-Sign-On

### ISSUE THREAD

**Issue by beriniwlew** [LibreTime/libretime#1181]
I would like to an easy way for my existing users to be able to log in to LibreTime using their existing credentials.

We're going to be using OpenID/OAuth for Single-Sign-On, and if LibreTime supported it, they wouldn't have to create brand new accounts.

I've considered AzuraCast, but I love the features of LibreTime

**Comment by paddatrapper** [comment]
This is definitely something we would like to support

**Comment by Robbt** [comment]
Yeah we have LDAP support but only documentation for FreeIPA and it would be great to support OAuth/OpenID as well.

**Comment by jooola** [comment]
Some ressources: https://github.com/mozilla/mozilla-django-oidc
